### PR TITLE
ci: skip k8s-oom test on ARM.

### PR DESCRIPTION
**Pull request by jongwu**
skip k8s-oom test on ARM as it can't pass on ARM for now,
even the root cause is unknown.

Fixes: #2887
Signed-off-by: Jianyong Wu <jianyong.wu@arm.com>

@devimc @jodh-intel

### CODE REVIEW

**Review by devimc — APPROVED** [review]
thanks @jongwu

**Review by GabyCT — APPROVED** [review]

**Comment by jodh-intel** [comment]
:tiger2: :elephant: :snake: :hippopotamus: :giraffe: :water_buffalo: :lion: :crocodile:

**Comment by jodh-intel** [comment]
Sorry @c3d - it's Monday. What can I say? :smile:

**Comment by jongwu** [comment]
> /lgtm but could you rephrase the commit comment slightly? I do not understand what you mean by "in the wild".
😆 , I write this words using my imagination.

**Review by bergwolf — APPROVED** [review]

**Comment by jongwu** [comment]
I'm confused with why can't I skip the k8s-oom test? any idea? @chavafg @jodh-intel

**Review by jodh-intel — APPROVED** [review]

File: integration/kubernetes/run_kubernetes_tests.sh [hunk 1]
```diff
@@ -66,6 +66,13 @@ if [ "${KATA_HYPERVISOR:-}" == "cloud-hypervisor" ]; then
 	oom_issue="https://github.com/kata-containers/tests/issues/2864"
 	info "$KATA_HYPERVISOR is failing on:"
 	info "pod oom: ${oom_issue}"
+
+elif [ "${arch}" == "aarch64" ]; then
```
Comment (jodh-intel) [inline integration/kubernetes/run_kubernetes_tests.sh:70]: Hmm, actually, it might be safer to check if the arch is *not* `aarch64` and if that is true, do `K8S_TEST_UNION+=("k8s-oom.bats")`. That avoids aarch64 getting out-of-sync if new tests are added and someone forgets to update *both* these blocks.

**Comment by jodh-intel** [comment]
Hi @jongwu - I'm not clear what you mean. The latest CI run for the "jenkins-ci-ARM-ubuntu-18-04" CI job shows:

```
09:28:52 Applying: ci: skip k8s-oom test on ARM.
```

**Comment by jongwu** [comment]
@jodh-intel - but the ci fails on k8s-oom test. 
`17:18:47 not ok 1 Test OOM events for pods`

**Comment by bergwolf** [comment]
@liubin also found something similar. Somehow CI is not checking out the correct branch since yesterday. @GabyCT @chavafg could you help to take a look?

**Comment by jodh-intel** [comment]
Ah - l think you're supposed to put the skips in https://github.com/kata-containers/tests/blob/master/.ci/aarch64/configuration_aarch64.yaml. Look at the block of code below the one you changed - that must be modifying the lists of tests to run.

**Comment by bergwolf** [comment]
In the mean time, this one can be merged IMHO.

**Comment by jongwu** [comment]
> Ah - l think you're supposed to put the skips in https://github.com/kata-containers/tests/blob/master/.ci/aarch64/configuration_aarch64.yaml. Look at the block of code below the one you changed - that must be modifying the lists of tests to run.

yeah, it's the right place. I post it again. I think this round will be OK. thanks

**Comment by jongwu** [comment]
@bergwolf - please give me another chance. I think the CI should not take the blame in this PR.
help retrigger the CI, @jodh-intel @devimc, thanks.

**Comment by chavafg** [comment]
@bergwolf @jodh-intel @jongwu  this change was the one that broke the CI: https://github.com/kata-containers/kata-containers/commit/c05c4ba5e743042f286cd049cdd63d9b0100d25b 
I opened https://github.com/kata-containers/kata-containers/pull/822 to fix this, ptal.

**Comment by jongwu** [comment]
Thanks @chavafg.

**Comment by jongwu** [comment]
@jodh-intel @bergwolf, I think we can neglect ARM CI and merge this PR, as there are two issues coupling with each other stop ARM CI to come back and we should fix them one by one. WDYT?

**Comment by jongwu** [comment]
Hello @jodh-intel @devimc, can't we merger this, As this PR is only related with aarch64.

**Comment by devimc** [comment]
I don't see the green button - retarting  `jenkins-ci-ubuntu-1804-cloud-hypervisor-k8s-containerd`

**Comment by jongwu** [comment]
hello @jodh-intel,  jenkins-ci-ubuntu-1804-cloud-hypervisor-k8s-containerd has passed. Is it time to merge this PR?

**Comment by bergwolf** [comment]
Is the PR working? I see that k8s-oom is still running for ARM. /cc @jongwu 
http://jenkins.katacontainers.io/job/kata-containers-2.0-ubuntu-ARM-PR/405/console

**Comment by jongwu** [comment]
It's strange, it should work. luckily, there is one success case, see [test409](http://jenkins.katacontainers.io/job/kata-containers-2.0-ubuntu-ARM-PR/409/)
